Client/Application/AccessibilityIdentifiers.swift

```diff
@@ -278,4 +278,10 @@ public struct AccessibilityIdentifiers {
     struct FindInPage {
         static let findInPageCloseButton = "FindInPage.closeButton"
     }
+
+    struct RememberCard {
```

Comment:
nit: Better to have credit card in the name so it makes it easier to find

```suggestion
    struct RememberCreditCard {
```